.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+# Deploys to Digital Ocean
+# After: https://medium.com/swlh/how-to-deploy-your-application-to-digital-ocean-using-github-actions-and-save-up-on-ci-cd-costs-74b7315facc2
+name: Deploy & Run
+on:
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy the app
+        uses: appleboy/ssh-action@v0.1.2
+        with:
+          host: ${{secrets.SSH_HOST}} # IP address of the server you wish to ssh into
+          key: ${{secrets.SSH_KEY}} # Public key of the server
+          username: ${{secrets.SSH_USERNAME}} # User of the server you want to ssh into
+
+          script: |
+            git clone https://github.com/OpenHospitalityNetwork/fedi-trustroots.git
```

Comment:
If this is running on a deployed server, then it should only `pull` not `clone`

Reply:
I see, gonna change it tomorrow morning.

In the article they use also ssh instead of https, would it make any difference for us?

Reply:
I changed it to `pull`